src/containers/Form/index.tsx

```diff
@@ -90,7 +84,7 @@ const Form = ({
     } else {
       setLoading(false);
       toggleSnackbar(true);
-      setSnackbarMessage("Subject and Claims are required fields.");
+      setSnackbarMessage("complite your claim,please!.");
```

Comment:
Why do you want to change the error message? The previous error message was clear. Subject and Claim are required fields. Others are not.

Reply:
I fixed it